geomstats/geometry/special_orthogonal.py

```diff
@@ -1,1664 +1,34 @@
-"""The special orthogonal group SO(n).
+"""Exposes the `SpecialOrthogonal` group class."""
```

Comment:
We should agree on a convention for docstrings at the top of the files. But right now, the implicit convention is not to be as concise as possible. Thus: `SpecialOrthogonal class.` would work. 

Being concise also allows avoiding inconsistency for example between:
"""Exposes the `SpecialOrthogonal` group class."""
and
"""Module exposing `Grassmannian` and `GrassmannianMetric` classes."""

Inconsistencies may not seem like a big deal, but on the automatic documentation website where all the .py files are listed together, it looks weird.